src/System.Device.Gpio/System/Device/Gpio/Drivers/RaspberryPiRegisterDriver.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Device.Gpio.Drivers;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace System.Device.Gpio
+{
+    internal class RaspberryPiRegisterDriver : GpioDriver
+    {
+        private GpioDriver _internalDriver;
+
+        public RaspberryPiRegisterDriver()
+        {
+            _internalDriver = new Windows10Driver();
```

Comment:
Since RaspberryPi3LinuxDriver derives from this (is that really intended?), this will throw on Linux.

Reply:
Never mind, this is already fixed in the next commit